build/flac-1.3.1/src/share/win_utf8_io/win_utf8_io_static.vcxproj

```diff
@@ -108,7 +109,6 @@
       <WholeProgramOptimization>true</WholeProgramOptimization>
       <AdditionalIncludeDirectories>.\include;..\..\..\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>WIN32;NDEBUG;_LIB;FLAC__NO_DLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
```

Comment:
Because the normal way to link things statically (at least according to conventional wisdom) is to use /MT.